hack/ci/e2e.sh

```diff
@@ -131,6 +130,11 @@ run_tests() {
 
 # setup kind, build kubernetes, create a cluster, run the e2es
 main() {
+    # ensure artifacts exists when not in CI
+    ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
```

Comment:
`${PWD}/_artifacts` should be `${WORKSPACE:$PWD}/_artifacts` I think?

Reply:
ARTIFACTS is set by both bootstrap and the podutils now, so I don't think WORKSPACE makes sense anymore